transformation/library/package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "transformation_linter",
+  "name": "transform_linter",
```

Comment:
this could be still transformation_linter. 
But it's a quick change for next times